test/functional/test_framework/util.py

```diff
@@ -322,6 +322,12 @@ def initialize_datadir(dirname, n):
 def get_datadir_path(dirname, n):
     return os.path.join(dirname, "node" + str(n))
 
+def append_config(dirname, n, options):
+    datadir = get_datadir_path(dirname, n)
+    with open(os.path.join(datadir, "bitcoin.conf"), 'a', encoding='utf8') as f:
```

Comment:
(32cc3fa349b56affb64b4a96c798228cdd9de083)

```suggestion
    with open(os.path.join(datadir, "pivx.conf"), 'a', encoding='utf8') as f:
```